ansible/roles/sat-ncn/defaults/main.yml.in

```diff
@@ -0,0 +1,38 @@
+#
+# MIT License
+#
+# (C) Copyright 2021-2022 Hewlett Packard Enterprise Development LP
+#
+# Permission is hereby granted, free of charge, to any person obtaining a
+# copy of this software and associated documentation files (the "Software"),
+# to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense,
+# and/or sell copies of the Software, and to permit persons to whom the
+# Software is furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included
+# in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+#
+# Default variables for sat-ncn role.
+---
+sat_repos:
+  - name: sat-sle-15sp2
+    # Really this is the URI, but the zypper_repository module uses "repo"
+    repo: "https://packages.local/repository/sat-sle-15sp2"
+    # Value replaced by runBuildPrep.sh
+    disable_gpg_check: @DISABLE_GPG_CHECK@
+    disable_repo_gpg_check: yes
+
+sat_packages:
+  - cray-sat-podman
```

Comment:
I'm not doing it here, but we would have the option of specifying the exact version of `cray-sat-podman` here, using variable substitution similar to what we're doing with `@SAT_VERSION@`.

The reason I considered doing this is that then we would have a new revision of configuration content when either `sat` or `sat-podman` changes version. Doing multiple installs does not change the commit hash of the VCS content we're importing unless something in that content has changed. This is mildly annoying since IMO you'd typically like batcher to kick off every time you run `update-mgmt-ncn-cfs-config.sh`.

Realistically I don't think this is a huge deal, though, since I expect `@SAT_VERSION@` to change more frequently. Additionally I think hard-coding the exact version of `cray-sat-podman` could be problematic because then old versions of the config would not be functional depending on what RPM(s) are actually available in the group repo.

Reply:
Your approach here seems reasonable. I think we can revisit this in the future if we need to.